.github/workflows/e2e-tests.yml

```
@@ -279,6 +279,10 @@ jobs:
         working-directory: ${{ matrix.path }}
         continue-on-error: true
         run: |
+          # Copy debug.log to output folder, if it exists
+          error_log="$GITHUB_WORKSPACE/tools/docker/wordpress/wp-content/debug.log"
+          [ ! -f $error_log ] || cp $error_log output/logs/php_errors.log
```

Some nitpicks:
* Best to use quotes, just in case there are odd characters.
* When running under bash, as Actions does by default, we may as well use `[[ ]]` rather than `[ ]`.

```suggestion
          [[ ! -f "$error_log" ]] || cp "$error_log" output/logs/php_errors.log
```

---

Thanks